argocd/argoapp-smallstep.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
ArgoCD app should be moved to here : https://github.com/advatys/kubernetes-library/tree/main/tlsaas/tls-cockpit

Reply:
removed